openff_sphinx_theme/openff_sphinx_theme/sass/site.sass

```
@@ -132,6 +132,9 @@ html
     @media screen and (min-width: $tablet) and (max-width: $desktop - 1px)
       margin-left: $mobile-content-margin-y
 
+article.content
+    max-width: $para-width
+    margin: auto
```

```suggestion
    @media screen and (min-width: $tablet) and (max-width: $desktop - 1px)
      margin-left: $mobile-content-margin-y
    @media screen and (min-width: $desktop)
      max-width: $para-width
      margin: auto
```

This is a good change, but we should only apply it above the desktop break point so that on smaller screens the content still has a margin (and doesn't go right to the edge of the screen). Thanks!